CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## unreleased
 
+## v4.3.0 - 2022.08.24
+* Update CSI driver for Kubernetes 1.24
+[[GH-457]](https://github.com/digitalocean/csi-digitalocean/pull/457)
+* Clarify release documentation on branch releases
```

Comment:
That's something we could exclude since that change does not impact the release artifact.

Reply:
I assume that you are referring only to l.6?

Reply:
yes @mo-rieger

Reply:
Yes.